rectools/metrics/debias.py

```diff
@@ -0,0 +1,204 @@
+#  Copyright 2024 MTS (Mobile Telesystems)
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Debias module."""
+
+import typing as tp
+from collections import defaultdict
+
+import attr
+import pandas as pd
+
+from rectools import Columns
+
+from .base import MetricAtK
+
+
+@attr.s(frozen=True)
+class DebiasConfig:
+    """
+    Config with debias method parameters.
+
+    Parameters
+    ----------
+    iqr_coef : float, default 1.5
+        Coefficient for defining as the maximum value inside the border.
```

Comment:
Let's please rewrite this to give more information, now it really doesn't help me to understand what is it used for.  At least let's add `interquartile range` phrase and some information of how it used for debiasing.  
Maybe also a little bit more information above to the description `DebiasConfig` itself. IMO now it's unclear what is it.

Reply:
Is that more understandable?

Reply:
Yes, perfect, thank you!